app/Console/Kernel.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace App\Console;
+namespace Findologic\Console;
```

Comment:
Changing the default namespace is discouraged by Laravel:

![image](https://user-images.githubusercontent.com/20555112/106722955-dbe53280-6606-11eb-9045-26416bab1f26.png)



See https://github.com/laravel/framework/pull/27575 for more information.